templates/stats.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{% import "macros.html" as macros %}
+{% block title %}Koschei - statistics{% endblock %}
+{% block content %}
+<div class="pageHeader">Koschei in numbers</div>
+<table class="details">
+    <tr><th>Stats update time</th><td>{{ stats.refresh_time }}</td></tr>
+    <tr><th>All packages</th><td>{{ stats.packages }}</td></tr>
+    <tr><th>Tracked packages</th><td>{{ stats.tracked_packages }}</td></tr>
+    <tr><th>Untracked packages</th><td>{{ stats.packages - stats.tracked_packages }}</td></tr>
+    <tr><th>Blocked packages</th><td>{{ stats.blocked_packages }}</td></tr>
+    <tr><th>Active packages</th><td>{{ stats.packages - stats.blocked_packages }}</td></tr>
+    <tr><th>All builds (real and scratch)</th><td>{{ stats.builds }}</td></tr>
+    <tr><th>Real builds</th><td>{{ stats.real_builds }}</td></tr>
+    <tr><th>Scratch builds</th><td>{{ stats.scratch_builds }}</td></tr>
+</table>
+
+<div class="pageHeader">Packages that consume most resources</div>
+{{ macros.pagination_row(page, 'Packages') }}
+<table class="data-list">
+    <tr class="list-header">
+        <th>Package</th>
+        <th>Arch</th>
+        <th>Time usage</th>
+        <th>Time percentage</th>
+    </tr>
+    {% for package in packages %}
+    <tr>
+        <td>
+            <a href="{{ url_for('package_detail', name=package.name)}}">
+                {{ package.name }}
+            </a>
+        </td>
+        <td>{{ package.arch }}</td>
+        <td>{{ package.time }}</td>
```

Comment:
Can you show it without miliseconds? They make it less readable and don't really add much value, since we're in the scale of days